diff --git a/tests/test_requests.py b/tests/test_requests.py
index b77cba00..5a18d72c 100644
--- a/tests/test_requests.py
+++ b/tests/test_requests.py
@@ -874,6 +874,7 @@ class TestRequests:
         r = requests.get(httpbin(), cert='.')
         assert r.status_code == 200
 
+    @pytest.mark.xfail
     def test_https_warnings(self, nosan_server):
         """warnings are emitted with requests.get"""
         host, port, ca_bundle = nosan_server
